spec/lib/terraform/runner_spec.rb

```diff
@@ -20,8 +20,10 @@
 
   describe "is .available" do
     before do
-      stub_request(:get, "#{terraform_runner_url}/ping")
-        .to_return(:status => 200, :body => {'count' => 0}.to_json)
+      ENV["TERRAFORM_RUNNER_URL"] = "https://1.2.3.4:7000"
```

Comment:
Is this needed since we do `stub_const("ENV", ENV.to_h.merge("TERRAFORM_RUNNER_URL" => terraform_runner_url))` up on line 11?